db/migrate/20200728054557_refer_config_proxy_to_http_proxy.rb

```diff
@@ -0,0 +1,17 @@
+class ReferConfigProxyToHttpProxy < ActiveRecord::Migration[6.0]
```

Comment:
Sorry I missed your question to me on IRC.

I think the proxy field should be `http_proxy_id` and reference the ID of an HttpProxy. So after we do the data migration you've done here, we can build the foreign key relationships, and remove the old `proxy` field